saltedge-compliance-connector-java-v2/compliance-connector-sdk/src/main/java/com/saltedge/connector/sdk/callback/mapping/SessionUpdateCallbackRequest.java

```diff
@@ -37,11 +37,20 @@
     @JsonProperty("session_expires_at")
     public Instant sessionExpiresAt;
 
+    @JsonProperty("funds_available")
+    public Boolean fundsAvailable;
+
     public SessionUpdateCallbackRequest() {
     }
 
     public SessionUpdateCallbackRequest(String redirectUrl, String status) {
         this.redirectUrl = redirectUrl;
         this.status = status;
     }
+
+    public SessionUpdateCallbackRequest(String redirectUrl, String status, Boolean fundsAvailable) {
```

Comment:
you don't have cases when redirectUrl and fundsAvailable exist in the same request